serveradmin/powerdns/models.py

```diff
@@ -1,3 +1,22 @@
 from django.db import models
 
-# Create your models here.
+
+class Domain(models.Model):
+    """PowerDNS Domain
+
+    Model to access domain in the PowerDNS db.
+    """
+    TYPES = [
+        ('MASTER', 'MASTER'),
+        ('SLAVE', 'SLAVE'),
+        ('NATIVE', 'SLAVE'),
+    ]
+
+    id = models.IntegerField(primary_key=True)
+    name = models.CharField(max_length=255, null=False)
+    master = models.CharField(max_length=128, default=None)
```

Comment:
What will be stored in here? Thought it would be a reference to the master then if it is a slave, but then I would assume a ForeignKey to Domain itself.

Reply:
I don't know yet. It is a given field of the PowerDNS [PostgreSQL schema](https://doc.powerdns.com/authoritative/backends/generic-postgresql.html#default-schema).

In our current database we don't use the field yet (it is null) so it might go away at a later point.

Maybe it is a good idea to link the schema for reference.

Reply:
👍🏼  for the schema reference linking!

Reply:
I linked it.